### PR TITLE
SEO-173271-Description-Too-Short-Core

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the description too short and image alt text changes.

Kind Regards,
Beryl.

### CODE REVIEW

File: aspnet-core/TreeGrid/Cells.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 title: Cells | TreeGrid | ASP.NET Core | Syncfusion
-description: cells
+description: Learn here about Cells in Syncfusion Essential ASP.NET Core TreeGrid Control, its elements, and more.
 platform: aspnet-core
 control: TreeGrid
 documentation: ug
 ---
 
-# Cell
+# Cells in TreeGrid ASP.NET Core
```
Comment (MallikaRK) [inline aspnet-core/TreeGrid/Cells.md:10]: Change the H1 like as
Cells in ASP.NET Core TreeGrid

Platform name should come first. Check and change in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/TreeGrid/Cells.md (hunk 1 above)
Comment (berylouma) [reply aspnet-core/TreeGrid/Cells.md]: Changed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**